crates/masonry/src/widget/label.rs

```diff
@@ -180,6 +186,13 @@ impl LabelMut<'_> {
         self.widget.alignment = alignment;
         self.ctx.request_layout();
     }
+
+    /// Set disabled
```

Comment:
What does it even mean for a label to be `disabled`?

Reply:
I guess it makes more sense in combination with a checkbox. But basically, that it's not clickable in HTML jargon at least

Reply:
Right, so e.g. the contents can't be clicked and dragged? I guess it also would have consequences for the accessibility metadata

Reply:
Yeah I guess that is indeed the case.